kitsune/sumo/email_utils.py

```diff
@@ -115,6 +115,7 @@ def make_mail(
         html = transform(
             render_email(html_template, context_vars),
             base_url="https://" + Site.objects.get_current().domain,
+            cssutils_logging_level=logging.ERROR,
```

Comment:
Why are you changing this from warning to error?

Reply:
Warning out puts a message every time that we are ignoring but suffering through for mso-table-rspace and mso-table-lspace. I have to leave them in the code because they are possibly in use, but there is no point in having cssutils cluttering the console about it.